resotocore/resotocore/cli/command.py

```diff
@@ -1889,6 +1897,20 @@ class ListCommand(CLICommand, OutputTransformer):
     a=aws_ec2_instance, b=sun
     a=aws_ec2_instance, b=moon
     a=aws_ec2_instance, b=star
+
+    # Properties that do not exist will be printed as empty values when using csv or markdown output.
+    > query is(aws_ec2_instance) limit 3 | list --csv kind as a, name as b, does_not_exist
+    a,b,does_not_exist
+    aws_ec2_instance,sun,
+    aws_ec2_instance,moon,
+    aws_ec2_instance,star,
+
+    > query is(aws_ec2_instance) limit 3 | list --markdown kind as a, name as b, does_not_exist
+    |a               |b   |does_not_exist|
```

Comment:
Consider to us a different example (is(aws_ec2_instance) will always yield the same kind)
```
> search is(instance) limit 3 | list instance_cores as cores, name, does_not_exist
```